bin/test.sh

```diff
@@ -63,12 +63,15 @@ configure_cli() {
 install_spark() {
     notify_github pending "Installing Spark"
 
-    # with universe server running, there are no longer enough CPUs to
-    # launch spark jobs if we give the dispatcher an entire CPU
-    # TODO: remove this?
-    echo '{"service": {"cpus": 0.1}}' > /tmp/spark.json
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
There's a separate job in jenkins that tests a given DC/OS build against package builds. It wouldn't be against the spark project CI but it'd be getting run periodically separately.